.github/actions/setup/action.yml

```diff
@@ -97,6 +97,7 @@ runs:
 
 
     - name: (windows) 7zip
+      shell: bash
```

Comment:
You don't need `shell:` when using `uses:`.

You only need `shell:` when using `run:`.

Reply:
:O